src/charm.py

```diff
@@ -113,7 +113,8 @@ def __init__(self, *args):
             self._ports.api,
             external_url=lambda: AlertmanagerCharm._external_url.fget(self),  # type: ignore
         )
-        self.api = Alertmanager(port=self._ports.api, web_route_prefix=self.web_route_prefix)
+
+        self.api = Alertmanager(endpoint_url=urljoin(self._external_url, self.web_route_prefix))
```

Comment:
I imagine we now need to push the cert to the charm container and run `update-ca-certificates` there as well.

Ref: https://github.com/canonical/grafana-k8s-operator/blob/a84d07e43c6c450950022663e2caac65f0918d70/src/charm.py#L1331

Reply:
do you mean [`_server_cert_path`](https://github.com/canonical/alertmanager-k8s-operator/blob/scheme_tls/src/charm.py#L81)?